src/engraving/dom/expression.cpp

```diff
@@ -48,6 +48,18 @@ Expression::Expression(const Expression& expression)
     _snapToDynamics = expression._snapToDynamics;
 }
 
+bool Expression::isEditAllowed(EditData& ed) const
+{
+    bool ctrlPressed  = ed.modifiers & ControlModifier;
+    bool shiftPressed = ed.modifiers & ShiftModifier;
+    bool altPressed = ed.modifiers & AltModifier;
+    if (altPressed && !ctrlPressed && !shiftPressed && (ed.key == Key_Left || ed.key == Key_Right)) {
+        return false;

```

Comment:
To be sure I understand it right: these changes are necessary so that the Expression itself doesn't consume the key press event, so that NotationActionController receives it instead, is that right?

Reply:
Yes exactly